qingstor/sdk/constant.py

```diff
@@ -23,3 +23,7 @@
 # Default MIME type
 JSON_MIME_TYPE = "application/json"
 BINARY_MIME_TYPE = "application/octet-stream"
+
+BLOCK_SIZE=32
+
+DEFAULT_IV=b"\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
```

Comment:
And this don't need to be in constant, other module is not likely to use this value.